api/ereqs_admin/static/admin/override.scss

```diff
@@ -7,6 +7,10 @@
 $banner-height: 60px;
 $nav-menu-width: 200px;
 
+* {
+  box-sizing: border-box;
```

Comment:
This subtly changes the admin -- I see it reducing the size of the content pane and slightly shortening the length of many text boxes.  Just to push back a little, are we sure this is necessary? I'm worried it'll break something unexpected.